package.json

```diff
@@ -17,6 +17,7 @@
     "lint": "eslint .",
     "test": "istanbul test -- _mocha",
     "sqlplus": "LD_LIBRARY_PATH=/opt/oracle/instantclient DYLD_LIBRARY_PATH=/opt/oracle/instantclient /opt/oracle/instantclient/sqlplus system/oracle@localhost:1521/xe",
+    "test-mssql": "LEDGER_UNIT_DB_URI=mssql://$MSSQL_USERNAME:$MSSQL_PASSWORD@in-1808.cojrajw6pfyj.us-west-2.rds.amazonaws.com:1433/$MSSQL_DATABASE node node_modules/.bin/istanbul test -- _mocha",
```

Comment:
We are running this on AWS because there is no docker image? I missed the discussion on why this is different than our setup for Oracle.